src/ssl.c

```diff
@@ -14223,6 +14236,16 @@ int wolfSSL_Cleanup(void)
     }
     session_lock_valid = 0;
     #endif
+    #ifdef SESSION_CACHE_DYNAMIC_MEM
+    for (i = 0; i < SESSION_ROWS; i++) {
+        for (j = 0; j < SESSIONS_PER_ROW; j++) {
+            if (SessionCache[i].Sessions[j]) {
+                XFREE(&SessionCache[i].Sessions[j], NULL, DYNAMIC_TYPE_SESSION);
```

Comment:
Does the `&SessionCache[i].Sessions[j]` need the leading `&`? Is heap hint not available here?